test/e2e/toolbox_test.go

```diff
@@ -371,10 +372,21 @@ var _ = Describe("Toolbox-specific testing", func() {
 
 		currentUser, err := user.Current()
 		Expect(err).To(BeNil())
+
 		session = podmanTest.Podman([]string{"run", "-v", fmt.Sprintf("%s:%s", currentUser.HomeDir, currentUser.HomeDir), "--userns=keep-id", fedoraToolbox, "sh", "-c", "echo $HOME"})
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
 		Expect(session.OutputToString()).To(ContainSubstring(currentUser.HomeDir))
+
+		location := path.Dir(currentUser.HomeDir)
+		volumeArg := fmt.Sprintf("%s:%s", location, location)
+		session = podmanTest.Podman([]string{"run",
+			"--userns=keep-id",
+			"--volume", volumeArg,
+			fedoraToolbox, "sh", "-c", "echo $HOME"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
```

Comment:
This line looks to be failing in the CI....

Reply:
Yes, the test doesn't seem to like being run as root.